static/js/partners.js

```diff
@@ -1,35 +1,44 @@
-/**
- * Generate a warnin only if warnings are available.
- */
-function warn(message) {
```

Comment:
It [looks like](http://caniuse.com/#search=console.warn) any calls to `console.*` in IE8 & IE9 will cause an error. Is that really acceptable? But maybe we should move it to `polyfills.js`.